shared-services/src/files/file-service.provider.ts

```diff
@@ -4,9 +4,10 @@ import { FileServiceInterface } from "./file-service.interface"
 import { FileConfig, FileServiceTypeEnum } from "./file-config"
```

Comment:
I think FileConfig should be renamed for clarity.  You will probably end up needing 4 different config types: 

- `CloudinaryFileServiceConfig` for Cloudinary-specific settings
- `AmazonS3FileServiceConfig` for S3-specific settings
- `FileServiceConfig` (which is just `CloudinaryFileServiceConfig | AmazonS3FileServiceConfig`)
- `FileProviderConfig`, which takes multiple `FileServiceConfig`s (see below)

Reply:
Done, added just the Cloudinary type for now and now the S3 one which will be in a future PR.